src/components/InfiniteAutoFill/InfiniteAutoFill.tsx

```diff
@@ -0,0 +1,25 @@
+import { type ReactElement, type RefCallback } from 'react';
+import { AutoFill } from '../../index.js';
+
+type InfiniteAutoFillChildrenProps = {
+  ref: RefCallback<unknown>;
+};
+
+type InfiniteAutoFillProps = {
```

Comment:
Can use the AutoFillProps instead of copying them, so this will be in sync if the others are updated.